orchestrator/app.py

```diff
@@ -207,6 +210,15 @@ def register_graphql(
         else:
             self.graphql_router.schema = new_router.schema
 
+    def register_authentication(self, authentication_instance: OIDCAuth) -> None:
+        self.auth_manager.authentication = authentication_instance
+
+    def register_authorization(self, authorization_instance: Authorization) -> None:
```

Comment:
Docs?

Reply:
Done!

Reply:
I found an empty file named `auth.md` in the docs folder, and I added a document to it. You can view the final version here:

https://github.com/workfloworchestrator/orchestrator-core/blob/feature/configurable-auth-handling/docs/reference-docs/auth.md

Is that what you want?